R/general_meanplot.R

```diff
@@ -81,20 +95,28 @@ general_meanplot <- function(data,
           strip.background = element_rect(fill = "grey90", color = "grey50"),
           plot.margin = margin(10, 10, 10, 10, "pt"))
 
-  ggplotly(p)
-
-  # add log scale
-  if (plot_ylog) {
-    p <- p + scale_y_log10()
-  }
   # add sd
   if (plot_sd) {
     p <- p +
       geom_errorbar(aes(ymin = (Mean - SD), ymax = (Mean + SD), color = id_variable), width = 0.4)
   }
 
-  return(p)
+  # add ci
+  if (plot_ci) {
+    p <- p +
+      geom_ribbon(aes(ymin = CI_lower, ymax = CI_upper, color = id_variable), alpha = 0.3)
+  }
+
+  # add log scale
+  if (plot_ylog) {
+    p <- p +
+      scale_y_log10(breaks = c(0.001, 0.01, 0.1, 1, 10, 100, 1000),
+                    label = c(0.001, 0.01, 0.1, 1, 10, 100, 1000)) +
+      annotation_logticks(sides = "l")
+  }
 
+  # Convert ggplot to plotly
+  return(p)
```

Comment:
nitpick: please use `return()` only for early returns

```suggestion
  p
```